### ISSUE TITLE
Question on visualization

### ISSUE THREAD

**Issue by hietwll** [AMReX-Codes/amrex#237]
Hello,
I have tried Paraview 5.4 and 5.5 for both linux and windows versions to open the simulation results (pltxxxxx/Header), but no reader named "Boxlib xxx" or "AMRex xxx" is found.

**Comment by ajnonaka** [comment]
Hi,

I just went to https://www.paraview.org/download/ and downloaded ParaView-5.5.0-RC4-Qt5-MPI-Linux-64bit.tar.gz

Then I did a "tar -xzvf ParaView-5.5.0-RC4-Qt5-MPI-Linux-64bit.tar.gz" and ran the paraview executable in the bin directory.  There was a Boxlib 3d option (see screenshot) and I was able to visualize the dataset as described in the Documentation.

![paraview](https://user-images.githubusercontent.com/10634549/38256848-0ea22bda-3714-11e8-8a27-fe5950410e4c.png)

Is this different from what you tried?

Andy

**Comment by JBlaschke** [comment]
Hi,

In case this helps, @ajnonaka pointed that you have to _manually_ type "Header" once you're  _in_ the plot file directory. Then paraview will prompt you about the file type, this is where you select "Boxlib 3D" This is very misleading (and also a problem when trying to open a time series of plot files, but I don't have a solution to that yet...)

I've updated the documentation to emphasize these points:
https://amrex-codes.github.io/amrex/docs_html/Visualization.html#paraview

**Comment by hietwll** [comment]
Hi,

The linux version "ParaView-5.5.0-RC4-Qt5-MPI-Linux-64bit.tar.gz" indeed works. I made a mistake that the paraview command in my system was linked to that of openfoam-paraview (version 5.4). It is not necessary to manually type "Header", just select "all files" → choose the Header file→ select “Boxlib 3d files”.

However, the Windows version still does not work, I have tested "ParaView-5.5.0-RC4-Qt5-MPI-Windows-64bit.exe", "ParaView-5.5.0-RC4-Qt5-Windows-64bit.zip", "ParaView-5.5.0-RC2-Qt5-Windows-64bit.exe", "ParaView-5.3.0-Qt5-OpenGL2-Windows-64bit.exe".

![windows-5-5rc4-1](https://user-images.githubusercontent.com/14847853/38290245-e2e9e6aa-380c-11e8-85f0-f3730826c20e.PNG)
![windows-5-5rc4-2](https://user-images.githubusercontent.com/14847853/38290251-eaa8d28e-380c-11e8-9359-e90a98c0ede4.PNG)

**Comment by emotheau** [comment]
For large datasets, Paraview is terribly slow to load in memory the solution file, as well as to perform any simple operation like cutting a domain into a plane. Maybe there are some options to tweak in the parameters?

**Comment by drummerdoc** [comment]
Obviously, this particular operation is something that amrvis3d is awesome at.  More generically though, there are a set of basic operations that require reading only a subset of the data in order to complete.  For such things, the best current option is to do the processing (slicing) with an AmrDerive-based tool, and then view the result afterward, but this is not great for interactive exploration.  I am piecing together a more interactive version of this time-honored tradition, via an extensible python interface along the lines of yt, but much more flexible, and tied closer to the parallel I/O capabilities of the computer centers.  Input/ideas to MSDay@lbl.gov for now.  I may start a pre-production input wiki or something.

**Comment by WeiqunZhang** [comment]
I think @zingale is also working on something similar.
https://github.com/AMReX-Astro/amrplot

**Comment by drummerdoc** [comment]
Ha!  Of course he is...

**Comment by ruihai0925** [comment]
@hietwll  I got the same problem a long time ago. That is what I asked: https://github.com/AMReX-Codes/amrex/issues/85

Another suggestion is to use Visit in the Linux system.  See here: https://wci.llnl.gov/simulation/computer-codes/visit/

I tried VisIt 2.13 Linux - x86_64 64 bit. It works well. If you use clusters for computing, most HPC systems will already install the Visit. So it is very convenient.

Good Luck!

Jordan
